EEGPSRCmdGen/Resources/eegpsr_cat2e.txt

```diff
@@ -87,16 +89,17 @@ $fgrefuse  = {void meta: The operator will actively refuse to follow/guide the r
 ##############################################################################
 $peopleR   = $ppl are $peopleDsc
 $peopleDsc = $posture | {gesture}
-$peopleDsc = wearing $outfit
-$peopleDsc = wear $color
+$peopleDsc = wearing $outfitss
+$peopleDsc = wearing $color
 
 $people    = $ppl
 $people    = $posture $ppl
```

Comment:
In the eegpsr_cat2.txt this case was changed to $ppl $posture is it on purpose that this wasn't changed here?

Reply:
The change to eegpsr_cat2.txt is correct and should have been done here as well. I have fixed this.